applications/DEMApplication/python_scripts/DEM_material_test_script.py

```diff
@@ -199,6 +202,7 @@ def CylinderSkinDetermination(self): #model_part, solver, DEM_parameters):
 
         h = self.height
         d = self.diameter
+        ymin = -0.000195748 #0.0
```

Comment:
hardcoded number?

Reply:
Yes, I was thinking about adding those values, ymin and zmin, through the ProjectParams.json. What do you think?

Reply:
Done